common/pb/admin_commands.proto

```diff
@@ -26,3 +27,13 @@ message Command_ReloadConfig {
 		optional Command_ReloadConfig ext = 1002;
 	}
 }
+
+message Command_AdjustMod {
+	extend AdminCommand {
+		optional Command_AdjustMod ext = 1003;
+	}
+	optional string admin_name = 1;
```

Comment:
In that case you definitely should get the admin name via the connection. If you use this, I can lie and said that a different user performed the action.